src/redis_cmd.cc

```diff
@@ -318,6 +380,48 @@ class CommandGet : public Commander {
   }
 };
 
+class CommandGetEx : public Commander {
+public:
+    Status Parse(const std::vector<std::string> &args) override {
+        if (std::find_if(args.begin(), args.end(), [](const std::string& str) -> bool {
+            return Util::ToLower(str) == "persist";
```

Comment:
`ToLower` will produce a new string, which is actually unnecessary for case-insensitive string comparison.

Reply:
You can define a new util function for case-insensitive string comparison.